eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -84,6 +87,20 @@ jobs:
 
           - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
 
+          - task:  UniversalPackages@0
+            displayName: Download ApiView Parser.
+            inputs:
+              command: download
+              vstsFeed: 'internal/AzureSDKForCpp'
+              vstsFeedPackage: 'parseazuresdkcpp'
+              downloadDirectory: '$(System.DefaultWorkingDirectory)/parser'
+              vstsPackageVersion: 0.6.0
```

Comment:
Is burning the package version here going to be a problem? Is there a way of specifying the latest version instead of tying to a specific version?

Reply:
We might be able to use it. I will check.

Reply:
You should be able to use wildcards here. If we're intentional about how breaking changes are versioned this will ensure that this job has to be modified here before a breaking version gets to be used. In this case a breaking change would be either 

* A change in the arguments required to run the CLI that needs modification here or 
* A change in the output format which requires a change in the way API view handles output

https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/universal-packages-v0?view=azure-pipelines

> vstsPackageVersion - Version
Input alias: versionListDownload. string. Required when internalOrExternalDownload = internal.
> 
> Specifies the package version or uses a variable containing the version to download. This entry can also be a wildcard expression, such as *, to get the highest version. Examples: 1.* gets the highest version with major version 1, and 1.2.* gets the highest patch release with major version 1 and minor version 2.

Reply:
Chatted offline w/ @praveenkuttappan and @LarryOsterman separately -- We'll keep this pinned today so we're at a "known good" version in our builds. Releases of later versions do require manual approval but we don't _need_ to float the version here today.